qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/helper/TestBase.java

```diff
@@ -471,4 +398,4 @@ private boolean checkIfPatientExists(String uuid) {
 		// TODO Auto-generated method stub
 		return false;
 	}
-}
+}
```

Comment:
any reason for removing the last empty line at the EOF?

Reply:
@kdaud it was about the IDE formatting but that I think its ouk

Reply:
An empty line at the end of the file caters for git collaborators so its's useful to maintain it across all the file.

Reply:
I think its ready for merge ....@kdaud data my suggestion:)